### ISSUE TITLE
Status Bar Should be White for better readability / contrast

### ISSUE THREAD

**Issue by frogg** [eu-digital-green-certificates/dgca-wallet-app-ios#53]
## Current Implementation
![IMG_2812](https://user-images.githubusercontent.com/5204169/118955172-bdf5d080-b95e-11eb-9689-f42e0976eda7.jpg)

## Suggested Enhancement

```swift
override var preferredStatusBarStyle: UIStatusBarStyle {
    return .lightContent
}
```

## Expected Benefits
Better accessibility, readability. Black on dark blue is not a great choice. See screenshot above.
